client/app/lib/changeteam/itemview.coffee

```diff
@@ -0,0 +1,67 @@
+kd      = require 'kd'
+JView   = require 'app/jview'
+globals = require 'globals'
+
+module.exports = class ChangeTeamListItem extends kd.ListItemView
+
+  JView.mixin @prototype
+
+  constructor: (options = {}, data) ->
+
+    options.cssClass = kd.utils.curry 'change-team-item', options.cssClass
+
+    super options, data
+
+    team = @getData()
+    { customize, slug } = team
+
+    logo = team.customize?.logo ? ''
+    @logoWrapper = new kd.CustomHTMLView
+      cssClass : 'team-logo-wrapper'
+    if logo
+      @logoWrapper.addSubView new kd.CustomHTMLView
+        tagName    : 'img'
+        attributes :
+          src      : "#{logo}"
+    else
+      @logoWrapper.hide()
+
+    { groupsController } = kd.singletons
+    currentGroup = groupsController.getCurrentGroup()
+
+    if currentGroup.slug is slug
+      @actionElement = new kd.CustomHTMLView
+        tagName  : 'span'
+        cssClass : 'active-team-label'
+        partial  : 'Active Team <div class="circle"></div>'
```

Comment:
change copy to `'Current Team'` also put the circle with a pseudo after element i.e. `.active-team-label:after`